src/app/fsat/explore-opportunities/explore-opportunities-form/explore-opportunities-form.component.ts

```diff
@@ -110,6 +110,9 @@ export class ExploreOpportunitiesFormComponent implements OnInit {
   }
 
   initForms() {
+    if (!this.fsat.modifications[this.exploreModIndex].exploreOppsShowVfd) {
```

Comment:
old assessments will have an fsat.isVFD value that should be used for the `hasOpportunity: false`